packages/crypto/src/sha.ts

```diff
@@ -10,3 +12,17 @@ export const sha256 = async (
   const hash = await mf.sha256.digest(bytes)
   return hash.digest
 }
+
+export const sha256Stream = async (stream: Readable): Promise<Uint8Array> => {
+  const hash = crypto.createHash('sha256')
```

Comment:
Here's what this might look like like using `stream.promises.pipeline()`:
```ts
  const hash = crypto.createHash('sha256')
  await pipeline(stream, hash)
  return hash.read()
```

Reply:
so i actually tried that, but it was getting backed up, blocking the stream, and then timing out 🤔 
so i ended up throwing it out & doing it this way lol

Reply:
Oh, funky!